### ISSUE TITLE
shift-tab converts tab indentation into spaces

### ISSUE THREAD

**Issue by SArpnt** [SthephanShinkufag/bytebeat-composer#37]
indentLess has to have a specific indentation specified, what should be used? i would go with tabs

**Comment by SthephanShinkufag** [comment]
Although I always use tabs in my code, I used double spaces everywhere in the formatted versions of big files because they make the code look more compact, especially on phones.
Should I replace double spaces with tabs in formatted versions?

**Comment by SArpnt** [comment]
i would recommend changing the tab width, and using them. personally i prefer a tab width of 3 since it's a reasonable balance of compactness and visibility

**Comment by SthephanShinkufag** [comment]
Good idea! Replaced two spaces into tabs for all formatted files, and made the size of tab for 3 spaces in codemirror. Now everything will be unificated, and there will be no mixing of styles.
